Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -52,6 +91,16 @@ public static int[] RemoveFirst(int[] array)
         public static int[] RemoveAt(int[] array, int index)
         {
             // ToDo: implement.
+            if(array != null || array.Length > 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['array' is null on at least one execution path.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)